components/ntp_background_images/browser/features.h

```diff
@@ -18,13 +18,13 @@ BASE_DECLARE_FEATURE(kBraveNTPSuperReferralWallpaper);
 
 BASE_DECLARE_FEATURE(kBraveNTPBrandedWallpaper);
 
-// Show initial branded wallpaper after nth new tab page for fresh installs.
+// Show initial branded wallpaper after nth new tab page for fresh opens.
 constexpr base::FeatureParam<int> kInitialCountToBrandedWallpaper{
-    &kBraveNTPBrandedWallpaper, "initial_count_to_branded_wallpaper", 1};
+    &kBraveNTPBrandedWallpaper, "initial_count_to_branded_wallpaper", 0};
```

Comment:
I would recommend leaving the default value and testing this initial count via this Griffin feature